cmd/influxd/config.go

```diff
@@ -34,11 +34,12 @@ const (
 
 // Config represents the configuration format for the influxd binary.
 type Config struct {
-	Hostname          string `toml:"hostname"`
-	BindAddress       string `toml:"bind-address"`
-	ReportingDisabled bool   `toml:"reporting-disabled"`
-	Version           string `toml:"-"`
-	InfluxDBVersion   string `toml:"-"`
+	Hostname              string `toml:"hostname"`
+	BindAddress           string `toml:"bind-address"`
+	ReportingDisabled     bool   `toml:"reporting-disabled"`
+	AuthenticationEnabled bool   `toml:"authentication-enabled"`
```

Comment:
That was exactly the way I coded first, before @pauldix sowed doubt in my mind and I reverted to this simpler form.